src/utils/client.ts

```diff
@@ -574,11 +574,31 @@ export default class Client {
     );
   }
 
-  createPayment(paymentMethodFieldKey: string) {
-    return this.payment('POST', { field_id: paymentMethodFieldKey });
+  createPayment() {
+    return this.payment('POST');
```

Comment:
maybe _payment since this is an internal method.

Reply:
also i can't coment directly on the payment method for some reason, but you could do

const data = { form_key: this.formKey, ...(userId ? ....), ...extraBodyParams};
const options = {body: data}

which is slightly simpler. also you do {method: method} but that can just be {method}

Reply:
also i think you may be able to assume userId always exists, i don't think we'd support payment actions without knowing who the user is.

Reply:
Eliminated extraBodyParams.